juntagrico/dao/subscriptionmembershipdao.py

```diff
@@ -18,3 +29,12 @@ def get_other_active_for_member(member, subscription, asof=None):
     @staticmethod
     def get_all_for_subscription(subscription):
         return SubscriptionMembership.objects.filter(subscription=subscription)
+
+    @classmethod
+    def current_of_members(cls, members):
+        return SubscriptionMembership.objects.filter(
+            cls.q_joined() & ~cls.q_left(),
+            member__in=members
+        ).annotate(
```

Comment:
we could do select related, but would then load the entire depot information instead of just the name.
I think for these optimizations we should assume the current use case and adapt them, if the use case changes.

Reply:
With #527 this annotation can/will be done only when we need it. I consider this an acceptable solution for the meantime.